Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSBeamCentreFinder.py

```diff
@@ -16,6 +16,7 @@
 from mantid.api import (DataProcessorAlgorithm, MatrixWorkspaceProperty, AlgorithmFactory, PropertyMode, Progress)
 from mantid.kernel import (Direction, PropertyManagerProperty, StringListValidator, Logger)
 from mantid.simpleapi import CloneWorkspace, GroupWorkspaces
+from sans.algorithm_detail.beamcentrefinder_plotting import can_plot_beamcentrefinder, plot_workspace_quartiles
```

Comment:
I don't think that something in Framework should depend on something in scripts. You should instead have the interface in scripts call this algorithm and the plotting separately

Reply:
@AndreiSavici A few algorithms do this as a way to consolidate code. It seems okay to me? Our issue really is that scripts is a bit of a mess of different bits of code.